history.txt

```diff
@@ -1,6 +1,39 @@
 OMERO version history
 =====================
 
+5.1.4 (September 2015)
+----------------------
+
+A bug-fix release covering all components. Improvements include:
+
+-  channel buttons fixed in OMERO.web
```

Comment:
Jason reported as a blocker saying it was unusable and I think anyone else who noticed it will immediately recognise what we're talking about so doesn't seem worth pushing another commit at this late stage. I'll give some extra detail in the release announcement instead.